### ISSUE TITLE
Invalid log level

### ISSUE THREAD

**Issue by yerlaser** [AndyObtiva/glimmer-dsl-xml#2]
I am trying the example on the homepage and get this before the actual html output

"Invalid log level:"

I'm on Fedora 35 with ruby 3.0.2p107

**Comment by AndyObtiva** [comment]
Which example? I just tried this one in IRB and it worked in ruby 3.0.2 (on Mac):

```ruby
require 'glimmer-dsl-xml'

include Glimmer

@html = html {
  head {
    meta(name: "viewport", content: "width=device-width, initial-scale=2.0")
  }
  body {
    h1 { "Hello, World!" }
  }
}

puts @html
```

Output:

```
<!DOCTYPE html><html><head><meta name="viewport" content="width=device-width, initial-scale=2.0"  /></head><body><h1>Hello, World!</h1></body></html>
```

I will try on Ubuntu Linux next.

**Comment by yerlaser** [comment]
Hi, yes, pretty much your example. Just tried commenting out some of my custom tags thus making markup similar to yours, but still get the same.

**Comment by AndyObtiva** [comment]
It works in Ubuntu Linux too:

IRB Input/Output:

```
3.0.2 :014 > puts @html
<!DOCTYPE html><html><head><meta name="viewport" content="width=device-width, initial-scale=2.0"  /></head><body><h1>Hello, World!</h1></body></html>
```

**Comment by AndyObtiva** [comment]
I'll try to install Fedora 35 and let you know how it goes with Ruby 3.0.2

By the way, I installed Ruby via [RVM](https://rvm.io)

**Comment by yerlaser** [comment]
Just tried exactly your example and still get the same.
So, it's either Fedora's Ruby or 3.0.2p107

**Comment by yerlaser** [comment]
OK. Thank you very much! It's not a big problem though, I can always grep that line out.

**Comment by AndyObtiva** [comment]
Nevermind, I've been able to replicate your problem even on Mac. It happens when running as an example rb file instead of inside IRB.

```
% ruby test1.xml.rb 
invalid log level: 
<!DOCTYPE html><html><head><meta name="viewport" content="width=device-width, initial-scale=2.0"  /></head><body><h1>Hello, World!</h1></body></html>
```

I'll look into it and get back to you.

**Comment by AndyObtiva** [comment]
OK, I have a hunch about this. In a recent version of the glimmer gem, which is a dependency of glimmer-dsl-xml, it started outputing an error message if the Glimmer Logger log level is incorrectly set. It is usually harmless in GUI DSLs like glimmer-dsl-swt, but it affects glimmer-dsl-xml negatively since every bit of output matters in it. I think fixing the issue is as simple as setting the right log level before loading the glimmer gem inside glimmer-dsl-xml. I'll get back to you once a patch is made for this. Thank you for reporting.

**Comment by AndyObtiva** [comment]
OK, my hunch was right. 

I just released a fix in [v1.2.1](https://rubygems.org/gems/glimmer-dsl-xml/versions/1.2.1).

**Comment by AndyObtiva** [comment]
I realized that upgrading the [glimmer](https://github.com/AndyObtiva/glimmer) gem to v2.4.1 also gets rid of that output (wihout needing extra code), so I just released [v1.3.0](https://rubygems.org/gems/glimmer-dsl-xml/versions/1.3.0) with a simpler solution (no glimmer logger level configuration code; just glimmer upgrade).

**Comment by yerlaser** [comment]
Yes, I confirm it works.
Thank you!
